services/network/public/cpp/content_security_policy/content_security_policy.cc

```diff
@@ -41,7 +41,12 @@ using DirectivesMap =
 namespace {
 
 bool IsDirectiveNameCharacter(char c) {
+#if BUILDFLAG(IS_COBALT)
+  // To accomodate "h5vcc".
```

Comment:
This is terrible. Could you add a TODO follow-up here so that we can migrate to a more sensible name please ? We already have `cobalt-insecure-...` , maybe we can implement `cobalt-location-src` here as an alternative name for this right away, so both names would be supported in client ? 

Then once server adds that name, we can remove this hack.